package.json

```diff
@@ -105,5 +112,11 @@
       "release": true,
       "tokenRef": "GITHUB_AUTH"
     }
+  },
+  "types": "index.d.ts",
+  "typeVersions": {
```

Comment:
:sob: wish I could check in types for .json files :sob:

Can you link me to the docs for this?

Reply:
Yeah, at a minimum schema validation would help. Docs are [here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html#version-selection-with-typesversions).

…and I'm glad you asked about this, because the key is *actually* `typesVersions`; I just typo'd it. 🤦‍♂️

Reply:
Awesome, thank you!